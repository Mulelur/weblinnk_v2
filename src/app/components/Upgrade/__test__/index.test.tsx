import * as React from 'react';
import { createRenderer } from 'react-test-renderer/shallow';

import { Upgrade } from '..';

const shallowRenderer = createRenderer();

describe('<Upgrade />', () => {
  it('should match snapshot', () => {
    shallowRenderer.render(<Upgrade />);
    const renderedOutput = shallowRenderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});
