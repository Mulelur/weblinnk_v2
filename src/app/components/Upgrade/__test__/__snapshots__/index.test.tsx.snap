// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Upgrade /> should match snapshot 1`] = `
<Upgrade__Container>
  <Upgrade__Content>
    <P>
      Upgrade Your Plan Now
    </P>
    <Upgrade__ButtonWarper>
      <styled.button
        onClick={[Function]}
      >
        Upgrade
      </styled.button>
    </Upgrade__ButtonWarper>
  </Upgrade__Content>
</Upgrade__Container>
`;
